### ISSUE TITLE
Use only Day view

### ISSUE THREAD

**Issue by atoyansk** [SundeepK/CompactCalendarView#9]
Hi SundeepK,

Your CompactCalendarView seems amazing widget to me! But I have a specific need to display it only into Day view. Is it possible?
If so, how can I do this?

Thanks a lot!

**Comment by SundeepK** [comment]
I'm not sure I understand the question. I couldn't find "Day view" in anything official for android. Do you want to hide and then show the widget like how google calendar works? Do you want to display events in a list format for a specific day? If so this library will never do that since its main focus is to show monthly views. Otherwsie, could you clarify a little bit more?

Thanks.

**Comment by atoyansk** [comment]
I need to display events only daily views, with its hours divisions... Just like this:
![image](https://cloud.githubusercontent.com/assets/12189750/9063851/07ae5704-3aa0-11e5-9524-e5da9e3ab4a6.png)

But, as I understood about your answer, it seems that is not possible, right?
Thanks!

**Comment by SundeepK** [comment]
Yes, this widget doesn't provide this functionality. Umm maybe a good idea for a new widget :).

**Comment by atoyansk** [comment]
I have several ideas about features for this widget. Would you like to build this widget on partnership? ;)

**Comment by SundeepK** [comment]
Actually I have written code that already does something similar to this but it's not open source yet. Are you familiar with google calendars? It has a week view calendar which shows exactly what you have shown in your image but in a week view. I wrote my own version which looks and acts almost like the one in the google app but there's only a few things left todo. The main thing is that It uses Joda time and I want to move that to using java Calendar so that there are no external dependencies. I haven't gotten round to doing that. 

The idea is that we can use that widget and add some extra code so that we can display x number of days into the screen. For example, in this case we can expose a method ```showNumberOfDays(int days)``` and set that to 1 and it will show a day view, currently it will show only 3 days(much how googles app does). I can spend some time this weekend/next week to try and make it open source but it will take time. Once I have the base code done, I will be happy to take pull requests to modify things. But no promise on the time line of things.

**Comment by atoyansk** [comment]
Perhaps the widget could display only one day at a time, but there would be a top navigation (using SlidingTab) that would display every day of the week. The current day's hours would be a list, which would be reloaded for the user slide. What do you think? It wolud be something like this:
![image](https://cloud.githubusercontent.com/assets/12189750/9088904/ddcf5720-3b68-11e5-8343-724275622314.png)

Others features that could be included (just ideas):
- Ability to work with data via JSON (both GET and PUT methods)
- Control event status (eg "Busy", "Queue", etc.)

**Comment by saddamandroid** [comment]
by default hide and show calendar on click
